apps/duck-web/src/mic.ts

```diff
@@ -0,0 +1,2 @@
+// Native ESM (pure) typescript, No ScriptProcessorNode
+import { clamp16, float32ToInt16 } from 'packages/duck-audio/src/pmc.js';\n\nexport type OnPcm = (pcm: Int16Array, tstamp?: number) => void;\n\nexport const startMic = async (onPcm: OnPcm): Promise<{ stop: () => Promise<void>; ctx: AudioContext }> => {\n  const ctx = new AudioContext({ sampleRate: 48000 });\n  await ctx.audioWorklet.addModule('/pcm16k-worklet.js');\n  const stream = await navigator.mediaDevices.getUserMedia({ audio: true });\n  const src = ctx.createMediaStreamSource(stream);\n  const node = new AudioWorkletNode(ctx, 'pcm16k');\n  node.port.onmessage = (evt) => {\n    const float = evt.data as Float32Array;\n    const pcm = float32ToInt16(float);\n    onPcm(pcm, Date.now());\n  };\n  src.connect(node);\n  // monitor silence: leaved disconnected to run headless\n  const stop = async () => {\n    node.disconnect();\n    src.disconnect();\n    stream.getTracks().forEach(t => t.stop());\n    await ctx.close();\n  };\n  return { stop, ctx };\n};\n
```

Comment:
Import path typo: `pmc.js` → `pcm.js`. Also prefer `performance.now()` for timestamps (monotonic), and reuse `float32ToInt16` from `@promethean/duck-audio` instead of duplicating clamp logic here.